justfile

```diff
@@ -1,4 +1,8 @@
 set positional-arguments
+alias t := cargo-tests
```

Comment:
Nice idea! Nit - should we alias these as `ct`, `cl`, `cf` to represent rust/cargo shortcuts? And we could have `gl` for go `lint-all`...

Reply:
I really like this idea yes